web_ui_gherkin.py

```diff
@@ -0,0 +1,212 @@
+import json
```

Comment:
I have started extracting some common UI code, so that we can have more of these scripts for specific agents, while sharing the common functionality. But I think separate `web_ui` scripts for each experiment are a way to go 👌